src/main/kotlin/no/nav/familie/tilbake/behandling/BehandlingService.kt

```diff
@@ -418,6 +422,7 @@ class BehandlingService(
         val eksternFagsakId = opprettTilbakekrevingRequest.eksternFagsakId
         val eksternId = opprettTilbakekrevingRequest.eksternId
         val erManueltOpprettet = opprettTilbakekrevingRequest.manueltOpprettet
+        var brevmottakere = opprettTilbakekrevingRequest.manuelleBrevmottakere
```

Comment:
Kan denne settes til `val` gitt at den ikke endres på ila denne funksjonen?
```suggestion
        val brevmottakere = opprettTilbakekrevingRequest.manuelleBrevmottakere
```